.circleci/config.yml

```diff
@@ -1,71 +1,56 @@
-# Based on config from github.com/alexeagle/angular-bazel-example
-version: 2
-
-# We use a docker image as the basis for our build, so that all the toolchains we use
-# are already installed and the build can start running right away. It also guarantees
-# the environment is portable and reproducible on your local machine.
-var_1: &docker_image angular/ngcontainer:0.10.0
-
-# Each job will inherit these defaults
-anchor_1: &job_defaults
-  working_directory: ~/ng
-  docker:
-    - image: *docker_image
-
-# After checkout, rebase on top of master, because we want to test the proposed merge of a
-# onto the target branch, not just test what's on the user's fork.
-# Similar to travis behavior, but not quite the same.
-# See https://discuss.circleci.com/t/1662
-anchor_2: &post_checkout
-  post: git pull --ff-only origin "refs/pull/${CI_PULL_REQUEST//*pull\//}/merge"
+version: 2.1
+commands:
+  install-bazel:
+    steps:
+      - run:
+          name: Install bazel
+          command: |
+            curl -OL https://github.com/bazelbuild/bazel/releases/download/0.24.0/bazel-0.24.0-installer-linux-x86_64.sh
+            chmod +x bazel-0.24.0-installer-linux-x86_64.sh
+            sudo ./bazel-0.24.0-installer-linux-x86_64.sh
+            rm ./bazel-0.24.0-installer-linux-x86_64.sh
+            bazel info release
+  restore-caches:
+    steps:
+      - restore_cache:
+          keys:
+            - "android_sdk"
+      - restore_cache:
+          keys:
+            - "v8-bazel_cache"
 
+  save-caches:
+    steps:
+      - save_cache:
+          key: "android_sdk"
+          paths:
+            - "/home/circleci/android_sdk"
 
-# These jobs will run in parallel, and report separate statuses to GitHub PRs
+      - save_cache:
+          key: "v8-bazel_cache"
+          paths:
+            - "/home/circleci/.cache/bazel/"
 jobs:
   lint:
-    <<: *job_defaults
+    working_directory: ~/ng/
```

Comment:
@oferb frankly speaking, because of my laziness to update `workspace_patcher`: https://github.com/google/startup-os/blob/d98b1eddc4989ef4f11d27d4914a4565df8cc9c6/tools/deps/WorkspacePatcher.java#L40

We can give it more generic name (i.e. `project`) and replace it both in `working_directory` and the line I linked above

Reply:
:)
Let's call it `code`.

Reply:
@oferb fixed in https://github.com/google/startup-os/commit/8bbd11a136c77a768a922fab21b6a9f544cf6ed1